textproc/heirloom-doctools/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        n-t-roff heirloom-doctools 160308
+categories          textproc
+platforms           darwin
+supported_archs     i386 x86_64
+license             CDDL
```

Comment:
It looks like most of the few other ports with this license are version 1. Can you verify the version?

`license              CDDL-1`

Reply:
The version used is 1.1. I did a quick grep and found both `CDDL` and `CDDL-1` so I went with the former. Tangential, but perhaps we should be using SPDX.

Reply:
If the version used is 1.1 then set it to:

`license CDDL-1.1`